spec/unit/mupdf_spec.lua

```diff
@@ -154,7 +155,10 @@ describe("mupdf module", function()
                 assert.equals(math.floor(bbox[1]*1000), 56145)
                 assert.equals(math.floor(bbox[2]*1000), 69233)
                 assert.equals(math.floor(bbox[3]*1000), 144790)
-                assert.equals(math.floor(bbox[4]*1000), 106089)
+                -- TBA: This dimension has changed with new mupdf,
+                -- possibly different font/rendering. Is that an issue?
+                --assert.equals(math.floor(bbox[4]*1000), 106089)
+                assert.equals(math.floor(bbox[4]*1000), 103669)
```

Comment:
@TnS-hun Just in case this ping works, is this to be expected or is something wrong here? (there's no visual difference, but who knows)